includes/core/ValueObjects/class-value-object.php

```diff
@@ -0,0 +1,86 @@
+<?php
+/**
+ * Create charge request.
+ *
+ * @package WooCommerce\Payments
+ */
+
+namespace WCPay\Core\DataTransferObjects;
+
+/**
+ * Base Value Object class.
+ */
+abstract class Value_Object {
```

Comment:
What's the difference between DTO and Value object? 
1. I genuinely want to raise discussion about the usage of these terms in this PR
2. Doc block doesn't give much away to help me

Reply:
This value object is moved to Request class for better understanding.